### ISSUE TITLE
[Bug] harmony player keeps telling me to enable plugins when i already did?

### ISSUE THREAD

**Issue by irockdude554** [vincelwt/harmony#215]
everytime I launch harmony now it tells me to enable plugins even though I already did that here's a link to a picture showing my issue https://i.imgur.com/4q9O425.png

**Comment by vincelwt** [comment]
Hey!

Thanks for reporting. Could you try quitting, resetting and restarting the app and see if the error persists?

From the faq:
![image](https://user-images.githubusercontent.com/5092466/38172904-e93e0b62-35b4-11e8-9bee-3456e18967b5.png)

**Comment by irockdude554** [comment]
that seemed to fix it for now, ill let you know if the problem comes back

**Comment by jackfranklin** [comment]
Hey,

Just to add a +1 here, I had this too. A complete reset has fixed it for now but I'll let you know if it happens again or if I figure anything out.

Thanks for your work on this, it's such a great little app :)

**Comment by vincelwt** [comment]
Thank you guys, please let me know if the problem comes back

**Comment by bsgreenb** [comment]
@vincelwt Reccomendation -- have reset settings clear out those cached locations

**Comment by vincelwt** [comment]
@bsgreenb I agree. Originally I thought that clearing settings would be enough for most problems but it's clear it's not.

**Comment by Izin** [comment]
@vincelwt your product seams to be awesome! And NOICE! I can pay my licence with cryptos??

Life is good

... Wait

I have this fuck*** plugin screen even if I clear cache, even if I restart, and even after I added Spotify and Deezer service?

--> Dude please, save me from removing your app just because of that... We are very happy that this app works with plugins. Sincerly...  But it is mandatory that you must ALWAYS give the ability to your users to close popup/modal windows/screens. And this feature must be clear, easy to understand and use.

You do that, I subs and I tell my friends about your nice concept.

Regards

**Comment by vincelwt** [comment]
@Izin Hey! Can you see your library behing the plugin screen or nothing?
